### PR TITLE
Add deleting user to bundle

### PR DESCRIPTION
**Pull request by jonasdekeukelaere**
Step one of integrating https://github.com/sumocoders/FrameworkMultiUserBundle/issues/44.
Step two will be in the framework itself.

### CODE REVIEW

File: Form/DeleteType.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+use SumoCoders\FrameworkMultiUserBundle\DataTransferObject\Interfaces\UserDataTransferObject;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class DeleteType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
```
Comment (StijnVrolijk) [inline Form/DeleteType.php]: : void

File: Form/DeleteType.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        $builder->add('id', HiddenType::class);
+    }
+
+    public function configureOptions(OptionsResolver $resolver)
```
Comment (StijnVrolijk) [inline Form/DeleteType.php]: : void

File: Form/DeleteType.php (hunk 1 above)
Comment (jonasdekeukelaere) [reply Form/DeleteType.php]: Fixed in rebase

File: Form/DeleteType.php (hunk 2 above)
Comment (jonasdekeukelaere) [reply Form/DeleteType.php]: Fixed in rebase

**Comment by jonasdekeukelaere** [comment]
Also added delete button to template in rebase

**Pull request merged**